frontend/www/js/omegaup/components/Countdown.vue

```diff
@@ -1,21 +1,50 @@
 <template>
-  <div class="base-timer">
-    <span class="base-timer__label">
+  <div>
+    <span>
       {{ formattedTimeLeft }}
     </span>
   </div>
 </template>
 
 <script lang="ts">
-import { Vue, Component, Prop } from 'vue-property-decorator';
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
 import * as time from '../time';
 
 @Component
 export default class Countdown extends Vue {
-  @Prop() timeLeft!: number;
+  @Prop() startTime!: Date;
```

Comment:
```suggestion
  @Prop() targetTime!: Date;
```

aquí se está haciendo el countdown hacia un tiempo final. el concepto de "tiempo inicial" es un detalle de implementación de este countdown cuando se usa en contexto de "esperar para que inicie el concurso", pero el mismo countdown debería poderse usar para "cuánto tiempo le queda a este concurso"?

Reply:
Listo, se cambia.